src/adapters/scene_category.js

```diff
@@ -27,13 +72,17 @@ export default class SceneCategory {
     listen('click_category_poi', state => {
       this.selectPoi(state);
     });
+    listen('clean_marker', () => {
+      this.selectPoiMarker(null);
+    });
+  }
+
+  getPointedPoi = mapMouseEvent => {
+    const feature = this.map.queryRenderedFeatures(mapMouseEvent.point)[0];
```

Comment:
It should be possible to use `mapMouseEvent.features` directly, instead of calling `queryRenderedFeatures()`.

Reply:
Thanks. I did that this way because the `mapMouseEvent.features` technique doesn't work for `mouseleave`… but I actually don't need to know the POI in this case.